usagov/usagov-2021
# Usagov 1855 homepage banner breakpoints override other breakpoints

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1855

## Description

Change the order of the homepage banner breakpoints to prevent the image from being overridden by other breakpoints. You can find the breakpoints in this file `web/themes/custom/usagov/templates/includes/homepage-banner.html.twig`.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Go to the homepage
2. Change the screen size to check that the homepage banner changes based on each of the following breakpoints:
      - @media(min-width: 640px)
      - @media(min-width: 720px)
      - @media(min-width: 960px)
      - @media(min-width: 1024px)
      - @media(min-width: 1280px)
      - @media(min-width: 1440px)
      - @media(min-width: 1536px)
      - @media(min-width: 1920px)
      - @media(min-width: 2048px)
      - @media(min-width: 3072px)
      
**_You can find the breakpoints in this file `web/themes/custom/usagov/templates/includes/homepage-banner.html.twig`._**

### Change Requirements



### Validation Steps

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
4. Find the commit of this pull request.
5. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
